### ISSUE TITLE
Sample project implementation

### ISSUE THREAD

**Issue by mrctito** [rdelrosario/xamarin-plugins#66]
Dear sir,

Do you have a sample project using CrossPushNotification implementation?

Although their explanations are very clear, I would be safer if I could compare my project with a sample project.

Thank you very much!
Best Regards,
Tito.

**Comment by treidio** [comment]
yes I would love a sample working project. I've been trying on and off for months to make this work.

**Comment by tyrtok** [comment]
would like to request for a sample project too

**Comment by Zalm** [comment]
Hi !
Does someone have a sample project or recent tutorial to share yet? I found some tutorials but they're all outdated :/
This would be awesome!
Thank you :)

**Comment by lucascc26** [comment]
+1 please

**Comment by HSregats** [comment]
I need an example too. I can't use this plugin :(

**Comment by Im-PJ** [comment]
A sample project please.... this looks like only such plugin available. So please update plugin to make it more robust
